src/web/app/pages-static/request-page/instructor-request-form/instructor-request-form.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { InstructorRequestFormComponent } from './instructor-request-form.component';
+
+describe('InstructorRequestFormComponent', () => {
+  let component: InstructorRequestFormComponent;
+  let fixture: ComponentFixture<InstructorRequestFormComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [InstructorRequestFormComponent],
+    });
+    fixture = TestBed.createComponent(InstructorRequestFormComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
Is it possible to test the form component more, like the [output](https://angular.io/guide/testing-components-scenarios#component-with-inputs-and-outputs)?